content/release-10-18-0/devices-10-18-0.md

```diff
@@ -94,6 +108,35 @@ Fix Version</th>
 <td>10.17.16.0</td>
 </tr>
 
+<tr>
+<td>Loriot</td>
+<td>Events created by the Loriot agent when sending uplink requests are now considered as device requests and provide connectivity status and availability monitoring.</td>
```

Comment:
```suggestion
<td>Events created by the Loriot agent when sending uplink requests are now considered device requests and provide connectivity status and availability monitoring.</td>
```

Reply:
I really stumble over this construction, but I trust you that it is correct. Sounds very weird to me which is why I left the "as" in:-).